components/Navigation/Navigation.js

```diff
@@ -0,0 +1,95 @@
+import Link from "next/link";
+import styled from "styled-components";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCat, faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { useRouter } from "next/router";
+
+// installed the fontawesome- react package
+// for my navigation icons.
+
+export default function Navigation() {
+  const router = useRouter();
+  const isCatOverview = router.pathname === "/";
+  const isFoodSearch = router.pathname === "/foodsearch";
+
+  return (
+    <>
+      <StyledContainer>
+        <StyledLink href={"/"}>
+          <StyledNavItem>
+            <FontAwesomeIcon
+              icon={faCat}
+              color={
+                isCatOverview
+                  ? "var(--navbar-icon-active)"
+                  : "var(--navbar-icon-inactive)"
+              }
+            />{" "}
+            <StyledNavText>Overview</StyledNavText>
+          </StyledNavItem>
+        </StyledLink>
+
+        <StyledLink href={"/foodsearch"}>
+          <StyledNavItem>
+            <FontAwesomeIcon
+              icon={faMagnifyingGlass}
+              color={
+                isFoodSearch
+                  ? "var(--navbar-icon-active)"
+                  : "var(--navbar-icon-inactive)"
+              }
+            />{" "}
```

Comment:
here as well :) sometimes its appended automatically by the editor

Reply:
I know what you mean. Yes, they were added automatically.
I don't need these and they are already deleted :) 
Thank you 😸